### PR TITLE
New Format and minify

### PR DESCRIPTION
**Pull request by MaanooAk**
Re-request #25

### CODE REVIEW

**Comment by wellsjo** [comment]
The output format function still uses `JSON.stringify`...I guess I kind of like that though since you get that vertical array view. What do you think?

I think the `Format` and `Minify` options should also be part of the context menu.

File: app/js/App.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ class App {
 
   constructor() {
     this.pages = [new Page()]
-	this.current_page = 0;
+    this.current_page = 0;
+
```
Comment (wellsjo) [inline app/js/App.js]: unnecessary whitespace

**Comment by MaanooAk** [comment]
Agree. About the context, these two options will be used once or twice on a session... and also in the context menu the format should check if some text is selected and be applied only on that.
TL;TR: Too complex, possibility in the feature.

**Comment by wellsjo** [comment]
Yeah, you make a good point. I made a small style fix, merging this now.
